python_ta/reporters/templates/template.html

```diff
@@ -4,111 +4,110 @@
+                        <div class="collapse">
+                            {% for indiv in file.style[message] %}
+                            <div class="fontbold">
+                                <span class="nbsp">
+                                [Line {{ indiv.line }}] {{ indiv.msg }}
+                                </span>
+                                {% if indiv.snippet != '' %}
+                                <pre>{{ indiv.snippet }}</pre>
+                                {% endif %}
+                            </div>
+                            {% endfor %}
+                        </div>
+                        {% else %}
+                        {{ no_err_message }}
+                        {% endfor %}
                     </div>
-                    {% endfor %}
-                </div>
-                {% else %}
-                {{ no_err_message }}
-                {% endfor %}
-            </div>
-            <div id="style">
-                {{ style_err_title }}
-                <div class="sliderall">
-                    <button> Expand All/Collapse All</button>
-                </div>
-            </div>
-            <div class="content collapse">
-                {% for message in file.style %}
-                <div class="info">
-                    <span class="style_id">
-                    <a href="http://www.cs.toronto.edu/~david/pyta/&#35;{{message}}" target="_blank">{{ message }} ({{ file.style[message][0].symbol }})</a>
-                    </span>
-                    <span class="nbsp">
-                    Number of occurrences: {{ file.style[message]|length }}
-                    </span>
-                    <span class="slider">
-                       <button> Expand/Collapse </button>
-                    </span>
-                </div>
-
-                <div class="collapse">
-                    {% for indiv in file.style[message] %}
-                    <div class="fontbold">
-                        <span class="nbsp">
-                        [Line {{ indiv.line }}] {{ indiv.msg }}
-                        </span>
-                        {% if indiv.snippet != '' %}
-                        <pre>{{ indiv.snippet }}</pre>
-                        {% endif %}
                     </div>
                     {% endfor %}
-                </div>
-                {% else %}
-                {{ no_err_message }}
-                {% endfor %}
+                    <a href="mailto:david@cs.toronto.edu">
+                        <div id="report_bug">
+                            Found a bug? Report it to Prof. Liu!
+                        </div>
+                    </a>
+                    <!--<script src="https://code.jquery.com/jquery-3.2.1.min.js"
+                        integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
+                        crossorigin="anonymous"></script>-->
+                    <script src="python_ta\reporters\templates\jquery-3.2.1.min.js"></script>
```

Comment:
Don't add changes that already exist in master. This applies to many other places as well.